### PR TITLE
Processor scheduling check

### PR DESCRIPTION
**Pull request by ramonsmits**
Added check to warn if processing scheduling isn't set to backgroundservices (reduces local SQL, Raven and MSMQ performance).

### CODE REVIEW

**Comment by ramonsmits** [comment]
@dvdstelt Please squash and not merge.

File: src/PerformanceTests/Common/Program.cs [hunk 1]
```diff
@@ -174,5 +175,16 @@ static void InitAppDomainEventLogging()
...
         }
+
+        static void CheckProcessorScheduling()
+        {
+            var key = Registry.LocalMachine.OpenSubKey(@"SYSTEM\CurrentControlSet\Control\PriorityControl");
+
+            if (key == null || 24 != (int)key.GetValue("Win32PrioritySeparation", 2))
+            {
+                Log.WarnFormat("Processor scheduling is set to 'Programs', consider setting this to 'Background services'!");
+                Thread.Sleep(3000);
```
Comment (dvdstelt) [inline src/PerformanceTests/Common/Program.cs]: I like this one, so many times ppl think I read their warnings ;-)

File: src/PerformanceTests/Common/Program.cs [hunk 2]
```diff
@@ -174,5 +175,16 @@ static void InitAppDomainEventLogging()
                 if (exception != null) unhandledLog.Error(exception.Message, exception);
             };
         }
+
+        static void CheckProcessorScheduling()
+        {
+            var key = Registry.LocalMachine.OpenSubKey(@"SYSTEM\CurrentControlSet\Control\PriorityControl");
+
+            if (key == null || 24 != (int)key.GetValue("Win32PrioritySeparation", 2))
```
Comment (dvdstelt) [inline src/PerformanceTests/Common/Program.cs]: Assuming you have the numbers right! :)

**Comment by dvdstelt** [comment]
Sorry, merged instead of squashed. Read the msgs after pressing the button :(

**Pull request merged**